config.yaml

```diff
@@ -30,6 +30,10 @@ options:
   mysql-interface-database:
     description: "The database name for the legacy 'mysql' relation"
     type: "string"
+  audit-plugin-enabled:
```

Comment:
Can we keep common with PostgreSQL naming?
[plugin_xxx_enable](https://charmhub.io/postgresql/configuration#plugin_vector_enable)